code/go/0chain.net/smartcontract/zcnsc/context_helpers_test.go

```diff
@@ -130,14 +105,14 @@ func (m *mockStateContext) GetTrieNode(key datastore.Key, v util.MPTSerializable
 	return util.ErrValueNotPresent
 }
 
-func MakeMockStateContext() *mockStateContext {
+func MakeMockStateContext() *mockStateContext { //nolint
```

Comment:
Not sure what it is being compained about here. But changing return type to state.StateContextI might fix all the other issues.

Reply:
We're not checking the lint errors on the `_test.go` files actually, there are too many warnings there so we won't fix them in this round.